generator/ServiceClientGeneratorLib/Operation.cs

```diff
@@ -321,15 +321,38 @@ public Member ResponsePayloadMember
         /// <summary>
         /// Gets the namespace of the payload for an XML object
         /// </summary>
-        public string XmlNamespace
+        public new string XmlNamespace
```

Comment:
I don't understand why we needed to add the `new` keyword here instead of simple override?

Reply:
Because I want them to hide the base methods so there isn't confusion between these and the base methods which are different. For example, without hiding them base.XmlNamespace could be still be used within this method incorrectly.